### ISSUE TITLE
Great work! Looking forward to the code

### ISSUE THREAD

**Issue by huixiancheng** [hustvl/AziNorm#1]
However, I have some ques about detail.
Q1: How to downsample points after overlapped patch splitting?
Q2: Why are there such big gaps in the performance of KPConv on the validation and test sets of SemanticKITTI in your paper? Based on your results on the validation set and the replicated experiments in other papers, **60± mIoU** should be the normal performance.

**Comment by outsidercsy** [comment]
Thanks for the attention and recognition to our work!
Q1: After patch splitting, we get a set of sub point clouds. Sub point clouds have the same data format and characteristics with the original point clouds, and can be processed in the same manner. How to downsample points is determined by the specific perception method, but not by AziNorm.
Q2:  Gaps may come from several aspects.  1) Different training schedules. 2) Different configurations. The authors provide different configurations of KPConv. 3) Different implementations. There exist many implementation reposities of  KPConv. Our experiments are based on this official PyTorch-based implementation (https://github.com/HuguesTHOMAS/KPConv-PyTorch). Perhaps other implementations are better.
By the way, the gap is not critical. Experiments based on KPConv are ablation studies about AziNorm. To validate AziNorm, we only need  to make sure that the ablation studies are conducted under the same experimental setting.
We are refactoring our code for better readability. And perhaps there would be a journal version of AziNorm, including thorough theoretical analyses and additional experiments. Thanks for the concern and please stay tuned.

**Comment by huixiancheng** [comment]
Thanks for your answer, looking forward to the final version of the paper and code.
